hc/front/tests/test_add_check.py

```diff
@@ -12,3 +12,12 @@ def test_it_works(self):
         assert Check.objects.count() == 1
 
     ### Test that team access works
+    def test_team_acess_works(self):
```

Comment:
- Make sure Bob and Alice are on the same team.
- Login as Alice and add a check
- Login as Bob and check if Bob has access to that check.